content/en/docs/ops/telemetry/in-proxy-service-telemetry/index.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Generate Istio Metrics Without Mixer
+description: In-proxy generation of service-level metrics.
+weight: 70
+---
+
+{{< warning >}}
+
+This document describes **Alpha** quality features. They have been exposed *only* to enable further testing
+and evaluation across the community.
+
+The functionality described here is NOT considered stable. No performance or stability guarantees are provided
+at this time.
+
+{{< /warning>}}
+
+Istio 1.3 adds experimental support to generate service-level HTTP metrics
+directly in the Envoy proxies. This feature lets you continue to monitor your service meshes using the tools Istio provides
+without needing Mixer. This support is the first step towards the new
+extensibility architecture for Istio. This architecture puts Mixer-like functionality directly in the Envoy proxies.
+
+The in-proxy generation of service-level metrics replaces the following HTTP metrics that Mixer currently
+generates:
+
+- `istio_requests_total`
+- `istio_request_duration_*`
+- `istio_request_size`
+
+## Enable Service-level Metrics Generation in Envoy
+
+To generate service-level metrics directly in the Envoy proxies, follow these steps:
+
+1.  To prevent duplicate telemetry generation, turn off the `istio-telemetry` service to disable calls to `istio-telemetry` in the mesh:
+
+
+    {{< text bash >}}
+    $ helm template install/kubernetes/helm/istio --name istio --namespace istio-system --set mixer.telemetry.enabled=false --set mixer.policy.enabled=false
+    {{< /text >}}
+
+        {{< tip >}}
+        Alternatively, you can comment out `mixerCheckServer` and `mixerReportServer` in your mesh's configuration file.
```

Comment:
not clear what you mean by "mesh configuration file" - provide an example or a link?

Reply:
There was a mention previously of the `istio` configmap. I guess that got removed in one of the suggestions/edits. I'll see if I can find a good reference link.

Reply:
Updated with a link to the reference doc.